jok.cpp

```diff
@@ -2,8 +2,8 @@
 int main()
 {
 	int i;
-		for(i=0;i<5;++i)
+		for(i=4;i<=5;++i)
```

Comment:
what do you want to do here?
This loop will only run for 2 values, i=4 and i=5. Also replace `++i` with `i++`.

Why `for` block has been indented? It should be in the same line as `i`.

Reply:
yes, you are write I have update it.